feature/bgp/static_route_bgp_redistribution/otg_tests/static_route_bgp_redistribution_test/static_route_bgp_redistribution_test.go

```diff
@@ -999,10 +1092,14 @@ func redistributeStaticRoutePolicyWithCommunitySet(t *testing.T, dut *ondatra.DU
 
 	policyStatementAction := policyStatement.GetOrCreateActions()
 	policyStatementAction.SetPolicyResult(oc.RoutingPolicy_PolicyResultType_ACCEPT_ROUTE)
-	policyStatementAction.GetOrCreateBgpActions().GetOrCreateSetCommunity().SetOptions(oc.BgpPolicy_BgpSetCommunityOptionType_ADD)
-	policyStatementAction.GetOrCreateBgpActions().GetOrCreateSetCommunity().GetOrCreateReference().SetCommunitySetRef(communitySetName)
-
+	if !deviations.BgpCommunitySetRefsUnsupported(dut) {
+		policyStatementAction.GetOrCreateBgpActions().GetOrCreateSetCommunity().SetOptions(oc.BgpPolicy_BgpSetCommunityOptionType_ADD)
+		policyStatementAction.GetOrCreateBgpActions().GetOrCreateSetCommunity().GetOrCreateReference().SetCommunitySetRef(communitySetName)
+	}
 	gnmi.Replace(t, dut, policyPath.Config(), redistributePolicyDefinition)
+	if deviations.BgpCommunitySetRefsUnsupported(dut) {
+		setCommunityViaCLI(t, dut, redistributeStaticPolicyName, policyStatementName, communitySetName)
+	}
```

Comment:
This needs to use  switch dut.Vendor() format.  See the requirements for deviations at:

https://github.com/openconfig/featureprofiles/blob/f80fb73ceb4b397b4c1cf6a7b373d1421650d0e9/internal/deviations/README.md?plain=1#L182-L187

(This is because the CLI you are adding is specific to Cisco.  Other vendors may also use this deviation, but with their own CLI)